client/extensions/woocommerce/state/action-types.js

```diff
@@ -1,3 +1,5 @@
+export const TAXRATES_REQUEST = 'TAXRATES_REQUEST';
```

Comment:
Agreed. Fixed in https://github.com/Automattic/wp-calypso/pull/15541/commits/17f50964f1f6156641032d53a6b96785ad2895a0